src/lib/db/feature-strategy-store.ts

```diff
@@ -399,7 +403,7 @@ class FeatureStrategiesStore implements IFeatureStrategiesStore {
         return mapRow(row[0]);
     }
 
-    private getAdminStrategy(
+    private static getAdminStrategy(
```

Comment:
Was the shift to static necessary for this? Don't have a problem with it, just interested